assets/js/hooks/session.js

```diff
@@ -157,6 +157,20 @@ const Session = {
       (event) => this.toggleCollapseAllSections(),
     );
 
+    this.subscriptions = [
+      globalPubsub.subscribe("jump_to_editor", ({ line, file }) => {
+        const cellId = file.split("#cell:");
```

Comment:
I think this doesn't work :D But to clarify, you can create `jumpToLine(file, line)` and share with this:

https://github.com/livebook-dev/livebook/blob/1f9229fe03dc717c856a9c6be5924cf5530eaf7c/assets/js/hooks/session.js#L564-L569